src/Sulu/Bundle/DomainEventBundle/Subscriber/PersistDomainEventSubscriber.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DomainEventBundle\Subscriber;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\DomainEventBundle\Entity\DomainEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+class PersistDomainEventSubscriber implements EventSubscriberInterface
```

Comment:
this implementation will flush thew changes to the database each time an event is emitted. i am not happy with this solution because it could flush unrelated changes and it might lead to a lot of flushes. not sure what would be better though - maybe we should collect all events and flush them at the end of the request?

Reply:
not sure but there is a possibility to flush a single entity - at least there was one in the passt